module/module.go

```diff
@@ -236,7 +236,7 @@ func NewModule(ctx context.Context, address string, logger logging.Logger) (*Mod
 	}
 
 	// attempt to configure PeerConnection
-	pcReady, pcClosed, err := rpc.ConfigureForRenegotiation(pc, rpc.PeerRoleServer, logger)
+	pcReady, pcClosed, err := rpc.ConfigureForRenegotiation(pc, rpc.PeerRoleServer, logging.GetOrNewLogger("rdk.network_outgoing"))
```

Comment:
some of these are a bit dubious as to whether or not they are incoming or outgoing network events in my opinion

Reply:
Why would we put this logger under `rdk.network_outgoing`? From what I can tell it's a logger used as part of connection establishment from a Go module back to the RDK. I think we should continue to use `logger`; that's the module's author's logger.

Reply:
discussed off PR, we are changing all `network_outgoing` / `network_incoming` to `networking`